lib/backend/dynamo/dynamodbbk.go

```diff
@@ -654,10 +657,17 @@ func (b *Backend) getTableStatus(ctx context.Context, tableName string) (tableSt
 // documentation in case users want to set up DynamoDB tables manually. Edit the
 // following docs partial:
 // docs/pages/includes/dynamodb-iam-policy.mdx
-func (b *Backend) createTable(ctx context.Context, tableName string, rangeKey string) error {
-	pThroughput := dynamodb.ProvisionedThroughput{
-		ReadCapacityUnits:  aws.Int64(b.ReadCapacityUnits),
-		WriteCapacityUnits: aws.Int64(b.WriteCapacityUnits),
+func (b *Backend) createTable(ctx context.Context, tableName string, rangeKey string, onDemand bool) error {
+	var pThroughput *dynamodb.ProvisionedThroughput
+	var billingMode *string
+	if onDemand {
```

Comment:
We don't need to pass this as a parameter, we don't do it for RCU/WCU.
```suggestion
	if b.OnDemand {
```